shared/src/commonTest/kotlin/com/example/tasky/commom/util/LongExtensionTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.tasky.commom.util
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import com.example.tasky.common.domain.util.toLocalDateTime
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlin.test.Test
+
+class LongExtensionTest {
+    @Test
+    fun `test toLocalDateTime`() {
+        val timeStamp = 1732642253000L
+        val actual = timeStamp.toLocalDateTime(TimeZone.UTC)
+
+        val expect = LocalDateTime(2024, 11, 26, 17, 30, 53)
+
+        assertThat(actual).isEqualTo(expect)
+    }
```

Comment:
Hmm 🤔 it looks like this test validates an external function `toLocalDateTime` from the kotlinx library. AFAIK these are pretty well tested already 😄 

Each unit test that you write adds more maintenance to your project - consider what tests would bring real value to your projects - testing custom implementations of your abstractions is a good place to start—also validating the behaviour of your viewmodels. These, for me, have more value than the above test.